s3storageprovider/src/main/java/org/duracloud/s3storage/S3StorageProvider.java

```diff
@@ -466,6 +467,11 @@ protected void doSetSpaceProperties(String spaceId,
         return map;
     }
 
+    @Override
```

Comment:
Overriding this method does not appear to serve any purpose, as it just calls the superclass method, which is already public.

Reply:
good catch.  I'm must have anticipated changes that didn't end up needing to be made.